Tasks/FileTransformV2/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "vsts-tasks-filetransform",
```

Comment:
check if version / name should be same as V1

Reply:
Yes it is same in all the other tasks